dom0-updates/qubes-dom0-update

```diff
@@ -173,8 +186,29 @@ elif [ -f /var/lib/qubes/updates/repodata/repomd.xml ]; then
         $guiapp
     else
         yum check-update
-        if [ $? -eq 100 ]; then
-            yum $YUM_OPTS $YUM_ACTION
+        if [ $? -eq 100 ]; then # Run yum with options
+
+            if [[ -n "$BAK_TEMPLATE_ROOT" ]] ; then
+            # Backup root.img just in case
+                echo -n "Renaming template root.img to root.img-bak..."
+                if mv "$BAK_TEMPLATE_ROOT" "$BAK_TEMPLATE_ROOT-bak" ; then
+                    echo "OK"
+                else
+                    echo; echo "ERROR: Could not rename root.img"
+                    exit 1
+                fi
+            fi
+
+            yum $YUM_OPTS $YUM_ACTION ; RETCODE=$?
+            
+            if [[ -n "$BAK_TEMPLATE_ROOT" ]] ; then
```

Comment:
Intention is to repeat the check at ln191: It means, "are we handling a template?"